streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -68,52 +92,224 @@ public void after() throws Exception {
     }
 
     @Test
-    public void testReprocessingFromScratchAfterResetWithoutIntermediateUserTopic() throws Exception {
-        super.testReprocessingFromScratchAfterResetWithoutIntermediateUserTopic();
+    public void shouldNotAllowToResetWhileStreamsIsRunning() {
+        final String appID = getTestId() + "-not-reset-during-runtime";
```

Comment:
A new safeUniqueTestName method was added to IntegrationTestUtils recently, can we port this class over to using that instead of hard-coding a semi-descriptive suffix to the appId in each test?